pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: atom_dartlang
-version: 0.6.0
+version: 0.6.1
```

Comment:
Can you drop this back down? We have a more complicated dance for rev'ing an atom plugin.